middlewares/sse.js

```diff
@@ -1,10 +1,19 @@
+const { hasAuthBearerHeader } = require('./apiToken');
+
 module.exports = function (req, res, next) {
+    function filterUser({ session, user }) {
+        const userId = session?.passport?.user || user?.id;
+        return req.user && req.user.id === userId;
+    }
+
     res.sse.sendTo = function (...args) {
-        req.session.save(() => {
-            this.send(({ session }) => {
-                return req.user && session.passport && req.user.id === session.passport.user;
-            }, ...args);
-        });
+        if (hasAuthBearerHeader(req)) {
+            this.send(filterUser, ...args);
+        } else {
+            req.session.save(() => {
+                this.send(filterUser, ...args);
+            });
+        }
```

Comment:
So, all messages are shared for one user for all clients? @blokhin check this, please.
I'm not sure this is good or just ok :)